clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryClient.java

```diff
@@ -59,99 +47,29 @@ public DataRecoveryMonitor getMonitor() {
-          continue;
-        }
-        try (ControllerClient parentCtrlCli = buildControllerClient(clusterName, url, params.getSSLFactory())) {
-          StoreHealthAuditResponse storeHealthInfo = parentCtrlCli.listStorePushInfo(s, false);
-          Map<String, RegionPushDetails> regionPushDetails = storeHealthInfo.getRegionPushDetails();
-          if (!regionPushDetails.containsKey(destFabric)) {
-            ret.put(s, Pair.of(false, "nothing to repush, store version 0"));
-            continue;
-          }
-          String latestTimestamp = regionPushDetails.get(destFabric).getPushStartTimestamp();
-          LocalDateTime latestPushStartTime =
-              LocalDateTime.parse(latestTimestamp, DateTimeFormatter.ISO_LOCAL_DATE_TIME);
-
-          if (latestPushStartTime.isAfter(timestamp)) {
-            ret.put(s, Pair.of(false, "input timestamp earlier than latest push"));
-            continue;
-          }
-
-          MultiStoreStatusResponse response = parentCtrlCli.getFutureVersions(clusterName, s);
-          // No future version status for target region.
-          if (!response.getStoreStatusMap().containsKey(destFabric)) {
-            ret.put(s, Pair.of(true, StringUtils.EMPTY));
-            continue;
-          }
-
-          int futureVersion = Integer.parseInt(response.getStoreStatusMap().get(destFabric));
-          // No ongoing offline pushes detected for target region.
-          if (futureVersion == Store.NON_EXISTING_VERSION) {
-            ret.put(s, Pair.of(true, StringUtils.EMPTY));
-            continue;
-          }
-          // Find ongoing pushes for this store, skip.
-          ret.put(s, Pair.of(false, String.format("find ongoing push, version: %d", futureVersion)));
-        }
-      } catch (VeniceException e) {
-        ret.put(s, Pair.of(false, "VeniceHttpException " + e.getErrorType().toString()));
-      }
-    }
-    return ret;
-  }
-
   public ControllerClient buildControllerClient(String clusterName, String url, Optional<SSLFactory> sslFactory) {
     return new ControllerClient(clusterName, url, sslFactory);
   }
 
   public void execute(DataRecoveryParams drParams, StoreRepushCommand.Params cmdParams) {
     Set<String> storeNames = drParams.getRecoveryStores();
-    if (storeNames == null || storeNames.isEmpty()) {
-      LOGGER.warn("store list is empty, exit.");
-      return;
-    }
-
-    Map<String, Pair<Boolean, String>> pushMap = getRepushViability(storeNames, cmdParams);
-    Set<String> filteredStoreNames = new HashSet<>();
-
-    for (Map.Entry<String, Pair<Boolean, String>> e: pushMap.entrySet()) {
-      if (e.getValue().getLeft()) {
-        filteredStoreNames.add(e.getKey());
-      } else {
-        this.getExecutor().getSkippedStores().add(e.getKey());
-      }
-    }
 
-    if (!filteredStoreNames.isEmpty()) {
-      if (!drParams.isNonInteractive && !confirmStores(filteredStoreNames)) {
+    if (!storeNames.isEmpty()) {
```

Comment:
`storeNames` could also be null, so let's use `StringUtils.isEmpty`, which include a null pointer check to avoid running into a NPE.